### DISCUSSION TITLE
Function trigger based on events in collection

### DISCUSSION THREAD

**Discussion by iamdineshkumar** [appwrite/appwrite#1469] [Q&A] [answered]
Hi is that possible to create function trigger based on collection specific even? 
For an ex. If i want to send an email to a client acknowledging to receipt of payment. In this case I just need to monitor one collection creation event and based on that can be send Email.

Note: This kind of feature i already used in parse server check out in thier documents for understanding
[https://docs.parseplatform.org/cloudcode/guide/#save-triggers](https://docs.parseplatform.org/cloudcode/guide/#save-triggers)

**Comment by stnguyen90** [comment]
Related [comment from Torsten](https://github.com/appwrite/appwrite/issues/1625#issuecomment-924368127):

> We were thinking about extending the events to the way we are managing channels with realtime.
> 
> https://appwrite.io/docs/realtime#channels
> 
> This way you would have more control. Not sure if this would fix the create/update problem, but it lets you narrow down for what resources functions are triggered in the first place.

**Comment by eldadfux** [comment]
The new events mechanism based on Appwrite's realtime events is already in the work and is expected to be released in the next version.

**Comment by stnguyen90** [comment, marked as answer]
As of [0.14.0](https://github.com/appwrite/appwrite/releases/tag/0.14.0), it's now possible to only trigger a function on a specific collection. For example, you would put `databases.[database ID].collections.[collection ID].documents.*.create` to trigger on each document created in database `[database ID]` and collection `[collection ID]`. For more info on the events you can hook to, refer to our [docs](https://appwrite.io/docs/webhooks#events).
